include/odp/api/spec/packet.h

```diff
@@ -1685,6 +1685,11 @@ odp_proto_l2_type_t odp_packet_l2_type(odp_packet_t pkt);
  *
  * Returns layer 3 protocol type. Initial type value is ODP_PROTO_L3_TYPE_NONE.
  *
+ * In addition to protocol types specified in ODP_PROTO_L3_TYPE_* defines,
+ * packet parser implementation may support parsing additional L3 protocol
+ * types. If protocol type is not recognized by parser, ODP_PROTO_L3_TYPE_NONE
+ * is returned.
```

Comment:
To be clear, we could mention here something like: "Returned protocol types values are from IEEE EtherType list, also those protocols that are not listed by ODP API."